### ISSUE TITLE
VSCode Extension - Internal Error

### ISSUE THREAD

**Issue by rmaclean** [fwcd/KotlinLanguageServer#21]
After resolving my previous issues ([19](https://github.com/fwcd/KotlinLanguageServer/issues/19) and [20](https://github.com/fwcd/KotlinLanguageServer/issues/20)), the extension seems to just randomly stop responding; either just doesn't work or offers suggestions for a few and then stops. No error message though the following does appear in the developer tools.

```/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2960 Error: Internal error.
	at new ResponseError (/Users/rmaclean/.vscode/extensions/fwcd.kotlin-0.1.3/node_modules/vscode-jsonrpc/lib/messages.js:46:28)
	at handleResponse (/Users/rmaclean/.vscode/extensions/fwcd.kotlin-0.1.3/node_modules/vscode-jsonrpc/lib/main.js:430:48)
	at processMessageQueue (/Users/rmaclean/.vscode/extensions/fwcd.kotlin-0.1.3/node_modules/vscode-jsonrpc/lib/main.js:258:17)
	at Immediate.<anonymous> (/Users/rmaclean/.vscode/extensions/fwcd.kotlin-0.1.3/node_modules/vscode-jsonrpc/lib/main.js:242:13)
	at runCallback (timers.js:672:20)
	at tryOnImmediate (timers.js:645:5)
	at processImmediate [as _immediateCallback] (timers.js:617:5)
```

If I take the code and build it myself and run it in debug it works just fine. I even built the extension manually and installed it, but then it stopped working again so it seems to be something about the communication between the language server and extension which talk fine when debugging but not when running normally.

The above all relate to my Mac; I have gotten this to reproduce on my Windows machine as well, by opening a folder. It seems happy if I just open a single file on Windows but haven't tried to that on MacOS or tried debugging the extension on Windows.

Any ideas or experiments I can run for you?

**Comment by fwcd** [comment]
@rmaclean Could you check, if there is a stacktrace of the language server itself under "Output -> Kotlin"?

![image](https://user-images.githubusercontent.com/30873659/41687115-dba00da4-74e6-11e8-929b-be371e775429.png)

(This is just a random stacktrace for demonstration purposes)

**Comment by rmaclean** [comment]
No Kotlin option in the drop down 😕

**Comment by fwcd** [comment]
@rmaclean Could you try updating the extension to the current published version and see if the issue persists?

**Comment by rmaclean** [comment]
Can confirm fixed on 0.15 🙌
